src/sparql/ordo-mapping-annotations-violation.sparql

```diff
@@ -0,0 +1,66 @@
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix IAO: <http://purl.obolibrary.org/obo/IAO_>
+prefix ECO: <http://purl.obolibrary.org/obo/ECO_>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix oio: <http://www.geneontology.org/formats/oboInOwl#>
+prefix def: <http://purl.obolibrary.org/obo/IAO_0000115>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix obo: <http://purl.obolibrary.org/obo/>
+prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+
+
+SELECT ?cls ?mapping_precision_string
+
+WHERE {
+  VALUES ?mapping { oboInOwl:hasDbXref }
+  ?cls a owl:Class;
+     	?mapping ?value .
+
+  	?xref_anno a owl:Axiom ;
+           owl:annotatedSource ?cls ;
+           owl:annotatedProperty ?mapping ;
+           owl:annotatedTarget ?value ;
+           ECO:0000218 ?mapping_precision_string .
+
+  FILTER( !isBlank(?cls))
+  
+    FILTER( !STRSTARTS(STR(?mapping_precision_string), "- NTBT (ORPHA code's Narrower Term maps to a Broader Term).\n- Inclusion term (The ORPHA code is included under a ICD10 category and has not its own code)."))
```

Comment:
I ran a test to make sure this worked as expected, and it did. I tried running it once as-is, and no results were returned, which is what we want. Then, I removed one of the `FILTER()` lines, ran it, and results were produced, which is also what I had expected.

Reply:
Wow man, good diligence.